src/test/resources/db/postgres/V6_1__adding_index_for_email_constraint.sql

```diff
@@ -0,0 +1,5 @@
+--CREATE UNIQUE INDEX deleted_not_null_idx ON professional_user (EMAIL_ADDRESS, DELETED)
+--WHERE DELETED IS NOT NULL;
+--
```

Comment:
Please remove comment code from the file

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/480/commits/dc4dc69e0df737dcc62fce07335d67d43f17c804